code/game/gamemodes/cult/cult_objectives.dm

```diff
@@ -163,6 +163,16 @@
 				possible_sac_targets += player.mind
 	return possible_sac_targets
 
+// Handles the updating of sacrifice objectives after the sacrifice target goes to cryo and ghosts
+/datum/game_mode/cult/proc/update_sac_objective(previous_target, previous_role)
```

Comment:
don't really need to take the previous_role variable, could just do previous_target.assigned_role but I guess this is neater

Reply:
Hm that is true